web-animations/interfaces/Animation/scroll-timeline-overallProgress.tentative.html

```diff
@@ -74,13 +75,18 @@
   // Verify initial start and current times once ready.
   assert_percents_equal(animation.currentTime, 0,
       "The current time is resolved when ready.");
+<<<<<<< HEAD:web-animations/interfaces/Animation/scroll-timeline-overallProgress.tentative.html
   assert_equals(animation.overallProgress, 0, "The overallProgress should be zero.");
+=======
+  assert_equals(animation.overallProgress, 0,
+    "The overallProgress should be zero.");
+>>>>>>> Rename Animation.progress to Animation.overallProgress:web-animations/interfaces/Animation/scroll-timeline-progress.tentative.html
```

Comment:
Looks there's an unresolved merge conflict here.

Reply:
Thanks, it has been fixed.